lib/msal-core/src/AuthResponse.ts

```diff
@@ -32,3 +32,15 @@ export function buildResponseStateOnly(state: string) : AuthResponse {
         accountState: state
     };
 }
+
+export function setResponseIdToken(originalResponse: AuthResponse, idToken: IdToken) : AuthResponse {
```

Comment:
Can we move this to a different file? Restricting AuthResponse.ts is for just the response type may be more ideal going forward. Create a new file if needed.

Reply:
I disagree, this is a helper for AuthResponse and naturally fits in this class. Too many files will start cluttering the workspace.

Reply:
Make ResponseUtils